javascript/src/main/pages/History/Instructor.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useState } from "react";
+import { Jumbotron } from "react-bootstrap";
+import { useAuth0 } from "@auth0/auth0-react";
+import { Redirect } from "react-router-dom";
+import BasicCourseTable from "main/components/BasicCourseSearch/BasicCourseTable";
+
+import CourseSearchFormQtrDeptOnly from "main/components/BasicCourseSearch/CourseSearchFormQtrDeptOnly";
+import CourseSearchFormInstructor from "main/components/BasicCourseSearch/CourseSearchFormInstructor";
+import JSONPrettyCard from "main/components/Utilities/JSONPrettyCard";
+import {  fetchBasicCourseHistoryJSON } from "main/services/courseSearches";
+import {  fetchInstructorHistoryNameQtrJSON } from "main/services/courseSearches";
+
+const Instructor = () => {
+
+    // every function that starts with "use" is a hook
+    // e.g. useState, useSWR, useAuth0
+
+    // courseJSON is the variable for the state
+    // setCourseJSON is the setter
+    // the parameter to useState is the initial value of the state
```

Comment:
I was told to remove these so you should too

Reply:
lol... yeah, these were here mostly to help folks when we were just starting out.   I agree that these can probably go.